packages/main/src/plugin/tasks/task-manager.ts

```diff
@@ -47,6 +50,21 @@ export class TaskManager {
       this.apiSender.send('toggle-task-manager', '');
       this.setStatusBarEntry(false);
     });
+
+    this.configurationRegistry.registerConfigurations([
+      {
+        id: 'preferences.experimental.tasks',
+        title: 'Tasks preference',
```

Comment:
Remove 'preference', it's redundant and none of our other preferences have it.

Actually, I am wondering if we should call the section Experimental? Have a single place to group any experimental preferences that we expect will disappear (become default behaviour or be rejected) or change ids in the future.

Reply:
https://github.com/containers/podman-desktop/issues/9236